interfaces/builtin/bool_file.go

```diff
@@ -112,8 +112,9 @@ func (iface *boolFileInterface) AppArmorConnectedPlug(spec *apparmor.Specificati
 }
 
 func (iface *boolFileInterface) dereferencedPath(slot *interfaces.ConnectedSlot) (string, error) {
-	if path, ok := slot.Attrs["path"].(string); ok {
-		path, err := evalSymlinks(path)
+	var path string
+	if err := slot.Attr("path", &path); err == nil {
```

Comment:
Not for this PR since this is just porting the code over, but this logic (both old and new) seems a bit awkward in terms of error handling. It magically pretends the value is not there if there are errors. Let's look into this again at some point.